workflows/microbiome/README.md

```diff
@@ -0,0 +1,89 @@
+# Microbiome Workflows
+
+The following workflows can be used directly for microbiome data analysis, pathogen detection, and tracking purposes. The workflows can also be adapted to any other sequencing technique.
```

Comment:
I've reviewed this file here, but I've noticed that the readme in the individual directories are duplicates of this text. Could you maybe just very briefly list the workflows here and describe how they relate to each other, then remove the copied text here so we only have to maintain one version of the text ?